ios/Classes/PspdfPlatformView.m

```diff
@@ -0,0 +1,116 @@
+//
+//  Copyright © 2018-2019 PSPDFKit GmbH. All rights reserved.
+//
+//  THIS SOURCE CODE AND ANY ACCOMPANYING DOCUMENTATION ARE PROTECTED BY INTERNATIONAL COPYRIGHT LAW
+//  AND MAY NOT BE RESOLD OR REDISTRIBUTED. USAGE IS BOUND TO THE PSPDFKIT LICENSE AGREEMENT.
+//  UNAUTHORIZED REPRODUCTION OR DISTRIBUTION IS SUBJECT TO CIVIL AND CRIMINAL PENALTIES.
+//  This notice may not be removed from this file.
+//
+#import "PspdfPlatformView.h"
+#import "PspdfkitFlutterHelper.h"
+#import "PspdfkitFlutterConverter.h"
+
+@import PSPDFKit;
+@import PSPDFKitUI;
+
+@interface PspdfPlatformView() <PSPDFViewControllerDelegate>
+@property int64_t platformViewId;
+@property (nonatomic) FlutterMethodChannel *channel;
+@property (nonatomic, weak) UIViewController *flutterViewController;
+@property (nonatomic) PSPDFViewController *pdfViewController;
+@property (nonatomic) UINavigationController *navigationController;
```

Comment:
Hmm... true we could go with that. If I change this now, I think the appearance customization KB example that I just added today won't work since we won't be embedded in a `UINavigationController` anymore. So I will need to update that.